classic_startup/python2.7libs/pythonrc.py

```diff
@@ -43,6 +44,26 @@ def classic_startup():
     # add to the system path
     sys.path.insert(0, tk_houdini_python_path)
 
+    # We need to check to make sure we don't have an incompatibility
+    # between httplib and Houdini's bundled ssl.py. This is a problem
+    # on some Linux distros (CentOS 7.x) with H16.
+    if sys.platform == "linux2" and sys.version.startswith("2.7.5"):
+        # We can check to see if ssl has the function we know that
+        # system httplib is likely to require. If it doesn't have it,
+        # then we need to force the use of our bundled httplib before
+        # we let the bootstrap happen.
+        import ssl
```

Comment:
Yes, you are. Unfortunately. httplib2 uses httplib, which calls an _ method on ssl.